### PR TITLE
fix clip extra when QAT export model

### PR DESCRIPTION
**Pull request by yghstill**
### PR types

Bug fixes
### PR changes

APIs
### Describe

When export model in QAT, `clip_extra=True`, which may cause some quantitative model fail to run normally on Paddle Lite. So reset `clip_extra=False`.

### CODE REVIEW

File: python/paddle/fluid/contrib/slim/tests/test_quantization_scale_pass.py [hunk 1]
```diff
@@ -173,7 +173,7 @@ def build_program(main, startup, is_test):
                 'quant_scale_model' + dev_name, ['image', 'label'], [loss],
                 exe,
                 server_program,
-                clip_extra=True)
+                clip_extra=False)
```
Comment (winter-wang) [inline python/paddle/fluid/contrib/slim/tests/test_quantization_scale_pass.py]: 单测可以不改吗？

File: python/paddle/fluid/contrib/slim/tests/test_quantize_transpiler_v2.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def build_program(main, startup, is_test):
                     './infer_model', ['image', 'label'], [loss],
                     exe,
                     test_program,
-                    clip_extra=True)
+                    clip_extra=False)
```
Comment (winter-wang) [inline python/paddle/fluid/contrib/slim/tests/test_quantize_transpiler_v2.py]: 单测可以不改吗

File: python/paddle/fluid/contrib/slim/tests/test_quantization_scale_pass.py (hunk 1 above)
Comment (yghstill) [reply python/paddle/fluid/contrib/slim/tests/test_quantization_scale_pass.py]: done

File: python/paddle/fluid/contrib/slim/tests/test_quantize_transpiler_v2.py (hunk 2 above)
Comment (yghstill) [reply python/paddle/fluid/contrib/slim/tests/test_quantize_transpiler_v2.py]: done

**Review by winter-wang — APPROVED** [review]
LGTM

**Pull request merged**